lib/handshake.ex

```diff
@@ -6,8 +6,12 @@ defmodule Membrane.DTLS.Handshake do
   fields:
   * client_mode :: boolean()
   * dtls_srtp :: boolean()
+  * parent :: pid()
 
-  Please refer to `ExDTLS` library documentation for meaning of these fields.
+  PID of parent is used to send cert fingerprint in form of `{:cert_fingerprint, fingerprint}`
+  where `fingerprint` is a binary.
```

Comment:
The parent pid shouldn't be passed explicitly. The fingerprint can be returned and then sent as a notification